src/jarabe/main.py

```diff
@@ -347,6 +347,51 @@ def setup_fonts():
     settings.set_property("gtk-font-name", "%s %f" % (face, size))
 
 
+def setup_proxy():
+    env_variables = ['http_proxy',
+                     'HTTP_PROXY',
+                     'https_proxy',
+                     'HTTPS_PROXY',
+                     'ftp_proxy',
+                     'FTP_PROXY',
+                     'socks_proxy',
+                     'SOCKS_PROXY']
+    g_mode = Gio.Settings('org.sugarlabs.system.proxy').get_string('mode')
+    if g_mode == 'manual':
+        schemas = ['org.sugarlabs.system.proxy.http',
+                   'org.sugarlabs.system.proxy.https',
+                   'org.sugarlabs.system.proxy.ftp',
+                   'org.sugarlabs.system.proxy.socks']
+        counter = 0
+        for schema in schemas:
```

Comment:
Can we just change this to:

    protos = ['http', 'https', ...]
    for proto in protos:
        name = '{}_proxy'.format(proto)
        schema = 'org.sugarlabs.whatever.{}'.format(proto)
        ...

would that be a little cleaner?